### PR TITLE
Fix EZP-24710: Paragraphs are stripped on the front-end if an XML blo…

### PR DESCRIPTION
**Pull request by glye**
…ck is published with the editor disabled

> Fixes https://jira.ez.no/browse/EZP-24710
> Status: Ready for review & ridicule

Ugly inefficient solution with hardcoded ezoe dependency, should not be merged as is, but seems to work fine. PR to make sure all tests pass.

### CODE REVIEW

**Comment by yannickroger** [comment]
Just in case you want to run the test locally, we have a prehistoric doc covering it http://www.ezpedia.org/ez/testing_ez_publish_test_system

**Comment by glye** [comment]
@yannickroger Yes, I know, just wanted to make sure all applicable tests are run (turns out there were not that many, given how fast it was).

**Comment by andrerom** [comment]
@dougplant unknown atm, ezxmltext handlers are one of the more complex parts of the system, and we need to solve it in the handler itself and not use oe handler for this.

File: kernel/classes/datatypes/ezxmltext/handlers/input/ezsimplifiedxmlinput.php [hunk 1]
```diff
@@ -105,6 +105,13 @@ function validateInput( $http, $base, $contentObjectAttribute )
             $contentObject->appendInputRelationList( $parser->getRelatedObjectIDArray(), eZContentObject::RELATION_EMBED );
             $contentObject->appendInputRelationList( $parser->getLinkedObjectIDArray(), eZContentObject::RELATION_LINK );
 
+            // Fixes EZP-24710 by processing the XML through ezoe
+            $tempOutputHandler = new eZXHTMLXMLOutput( $xmlString, false, contentObjectAttribute );
```
Comment (joaoinacio) [inline kernel/classes/datatypes/ezxmltext/handlers/input/ezsimplifiedxmlinput.php]: should `contentObjectAttribute` be a var?

Comment (glye) [inline kernel/classes/datatypes/ezxmltext/handlers/input/ezsimplifiedxmlinput.php]: Nope. Fixing...

**Comment by dougplant** [comment]
Ok, thanks for that.

**Pull request closed without merging**